TasksMeter.c

```diff
@@ -29,9 +29,8 @@ static void TasksMeter_updateValues(Meter* this) {
    this->values[1] = pl->userlandThreads;
    this->values[2] = pl->totalTasks - pl->kernelThreads - pl->userlandThreads;
    this->values[3] = MINIMUM(pl->runningTasks, pl->activeCPUs);
-   this->total     = pl->totalTasks;
 
-   xSnprintf(this->txtBuffer, sizeof(this->txtBuffer), "%d/%d", (int) this->values[3], (int) this->total);
+   xSnprintf(this->txtBuffer, sizeof(this->txtBuffer), "%d/%d", (int) this->values[3], pl->totalTasks);
```

Comment:
```suggestion
   xSnprintf(this->txtBuffer, sizeof(this->txtBuffer), "%d/%d", MINIMUM(pl->runningTasks, pl->activeCPUs), pl->totalTasks);
```

Reply:
The data types of the argument is `unsigned int`, so it would be more correct to change the format string as well.
```suggestion
   xSnprintf(this->txtBuffer, sizeof(this->txtBuffer), "%u/%d", MINIMUM(pl->runningTasks, pl->activeCPUs), pl->totalTasks);
```
Will incorporate this in the next rebase.